### ISSUE TITLE
Image is lost during export if "Name" is not unique

### ISSUE THREAD

**Issue by Tanarri** [joyfullservice/msaccess-vcs-addin#268]
Hi,

I found an issue on exporting the images from table "MSysResources".

Reproducible with:
insert 2 records in table "MSysResources":
Attachment: test.png / Extension: png / ID: 1 / Name: test / Type: img
Attachment: test.ico / Extension: ico / ID: 2 / Name: test / Type: img

Please note we can insert 2 Items with th same "name" but different image-attachments.
Then when exporting, only one "test.json" but 2 images (test.png + test.ico) will be written to disk.

I have not testet the import, but this will result to wrong data on import.

Idea to resolve this issue:
Perhaps we can write 2 "Items" in the JSON file and improve the import.

Greetings
TanarRi

**Comment by joyfullservice** [comment]
Interesting. Thanks for pointing this out! For consistency, I kind of like the idea of using a separate file to represent each image, but maybe we should also include the extension in the file name for the export file.

`test_png.json` would then be different from `test_ico.json`, and both files would be imported as expected.

**Comment by Tanarri** [comment]
I have verified the test of @joyfullservice and he is right - the described behavior also exists in access 2013.

I agree that the program behave correctly with the actual implementation. If the developer manually make changes in the table 'MSysRecource' it's out of scope this project.

Thanks for all the info. I have sometimes directly made changes in the table, which should not be done, and thus put an error there myself. After carefully searching my records in 'MSysResource' I found the error and corrected it.
In the future I will only upload and maintain images in this table using the Access function.

I think we can close this issue.
